app/controllers/restaurant_controller.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+class RestaurantController < ApplicationController
+
+  before_action :set_cuisines, :except => [:show, :delete, :destroy]
+
+  def index
+    @restaurants = Restaurant.sorted
+  end
+
+  def show
+    @restaurant = Restaurant.find(params[:id])
+  end
+
+  def new
+    @restaurant = Restaurant.new({:name => 'Default'})
+  end
+
+  def create
+    @restaurant = Restaurant.new(restaurant_params)
+    if @restaurant.save
+      logger.debug "****** New restaurant was seved to DB. *******"
```

Comment:
Typo: `saved to DB`

Also, please remove logger.debug from production code :)

Reply:
Done